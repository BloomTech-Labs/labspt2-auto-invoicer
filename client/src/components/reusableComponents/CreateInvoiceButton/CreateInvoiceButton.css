.invoice-card .add-icon {
  font-size: 60px;
}

.add-invoice:hover {
  border-color: darkgray;
}
