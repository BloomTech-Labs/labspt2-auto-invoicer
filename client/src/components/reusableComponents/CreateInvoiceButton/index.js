// import packages
import React, { Component } from "react";
import AddIcon from "@material-ui/icons/AddCircle";

// import styles
import "./CreateInvoiceButton.css";

class CreateInvoiceButton extends Component {
  render() {
    return (
      <article className="invoice-card add-invoice">
        <h3>New Invoice</h3>
        <AddIcon className="add-icon" />
      </article>
    );
  }
}

export default CreateInvoiceButton;
