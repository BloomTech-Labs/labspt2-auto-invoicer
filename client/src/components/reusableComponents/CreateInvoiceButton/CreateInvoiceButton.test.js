import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import CreateInvoiceButton from "../CreateInvoiceButton";

describe("<CreateInvoiceButton />", () => {
  it("renders Create Invoice Button", () => {
    const { asFragment } = render(<CreateInvoiceButton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
