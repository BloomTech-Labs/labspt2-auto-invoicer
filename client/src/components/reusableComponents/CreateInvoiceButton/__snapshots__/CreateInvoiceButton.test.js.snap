// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateInvoiceButton /> renders Create Invoice Button 1`] = `
<DocumentFragment>
  <article
    class="invoice-card add-invoice"
  >
    <h3>
      New Invoice
    </h3>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root-1 add-icon"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm5 11h-4v4h-2v-4H7v-2h4V7h2v4h4v2z"
      />
    </svg>
  </article>
</DocumentFragment>
`;
